jib-core/src/main/java/com/google/cloud/tools/jib/registry/RegistryAuthenticator.java

```diff
@@ -254,18 +255,21 @@ public Authorization authenticatePush() throws RegistryAuthenticationFailedExcep
 
   @VisibleForTesting
   URL getAuthenticationUrl(String scope) throws MalformedURLException {
-    StringBuilder urlBase = new StringBuilder(authenticationUrlBase);
-    urlBase
+    StringBuilder urlBuilder = new StringBuilder(authenticationUrlBase);
+    urlBuilder
         .append("&scope=repository:")
         .append(registryEndpointRequestProperties.getImageName())
         .append(':')
         .append(scope);
 
-    for (String repository : registryEndpointRequestProperties.getDependentImageNames()) {
-      urlBase.append("&scope=repository:").append(repository).append(":pull");
+    if (registryEndpointRequestProperties.getMountedImageName() != null) {
+      urlBuilder
```

Comment:
To actually authenticate for the pull, I believe we would need to add the authentication credentials for the base image.